packages/flutter/lib/src/widgets/view.dart

```diff
@@ -204,7 +263,7 @@ class View extends StatelessWidget {
 /// Used by [_RawView.builder].
 typedef _RawViewContentBuilder = Widget Function(BuildContext context, PipelineOwner owner);
 
-/// The workhorse behind the [View] widget that actually bootstraps a render
+/// The workhorse behind the [RawView] widget that actually bootstraps a render
```

Comment:
The PR description (and this doc) promised the introduction of a RawView widget, but looks like that wasn't implemented? I think it would still be good to have a `RawView` that only deals with the pure rendering aspects and outsources focus (and probably also MediaQuery) management to the higher-level `View` widget.

Reply:
Oh, whoops. Yes, that was a U-turn that I did when making this. I thought I reverted them all, but I guess not.

But if you want the separation, I can put it back. :-)

Reply:
What made you do a U-turn? Since technically, the focus system is optional I think it would be still valuable to have that separation, unless there was a good reason not to do it?

Reply:
Mainly just ease of implementation: I could make the state a `WidgetsBindingObserver` and keep the scope node local. Mostly I didn't need to have it be separate for it to work. I can still separate it out though if you'd rather it be that way for separation of purpose (which does make sense).

Reply:
I think we should separate these out.